cmake/modules/hunter_get_toolchain_binaries.cmake

```diff
@@ -48,7 +48,10 @@ function(hunter_get_toolchain_binaries)
         CMAKE_C_PREPROCESSOR
         "${CMAKE_C_ANDROID_TOOLCHAIN_PREFIX}cpp${CMAKE_C_ANDROID_TOOLCHAIN_SUFFIX}"
     )
-    if(NOT EXISTS "${CMAKE_C_PREPROCESSOR}")
+    # GCC has been removed from NDK r18.
+    if(${ANDROID_NDK_MAJOR} VERSION_GREATER 17)
+      set(PARAM_OUT_CPP "")
```

Comment:
By making this variable empty you trigger the behavior like user haven't requested for `CPP` parameter. I think it's more accurate will be to set output variable to empty value instead (user requested for variable and result is empty string). In this case this line should be removed:
* https://github.com/ruslo/hunter/blob/c6cf6144e1df345ec8ef1dd2b73b67da34fed702/cmake/modules/hunter_get_toolchain_binaries.cmake#L60

Reply:
As a final result it will be:
```cmake
if(NOT EXISTS "${CMAKE_C_PREPROCESSOR}")
  # GCC has been removed from NDK r18.
  set(CMAKE_C_PREPROCESSOR "")
endif()
```